pkg/stages/steps_init.go

```diff
@@ -152,39 +153,10 @@ func GetKairosReleaseStage(sis values.System, log types.KairosLogger) []schema.S
 		"KAIROS_INIT_VERSION":   values.GetVersion(),                                 // The version of the kairos-init binary
 	}
 
-	// Get SOFTWARE_VERSION from the k3s/k0s version
-	if config.DefaultConfig.Variant == config.StandardVariant {
-		log.Logger.Debug().Msg("Getting the k8s version for the kairos-release stage")
-		var k8sVersion string
-
-		switch config.DefaultConfig.KubernetesProvider {
-		case config.K3sProvider:
-			out, err := exec.Command("k3s", "--version").CombinedOutput()
-			if err != nil {
-				log.Logger.Error().Msgf("Failed to get the k3s version: %s", err)
-			}
-			// 2 lines in this format:
-			// k3s version v1.21.4+k3s1 (3781f4b7)
-			// go version go1.16.5
-			// We need the first line
-			re := regexp.MustCompile(`k3s version (v\d+\.\d+\.\d+\+k3s\d+)`)
-			if re.MatchString(string(out)) {
-				match := re.FindStringSubmatch(string(out))
-				k8sVersion = match[1]
-			} else {
-				log.Logger.Error().Msgf("Failed to parse the k3s version: %s", string(out))
-			}
-		case config.K0sProvider:
-			out, err := exec.Command("k0s", "version").CombinedOutput()
-			if err != nil {
-				log.Logger.Error().Msgf("Failed to get the k0s version: %s", err)
-			}
-			k8sVersion = strings.TrimSpace(string(out))
-		}
-
-		log.Logger.Debug().Str("k8sVersion", k8sVersion).Msg("Got the k8s version")
-		env["KAIROS_SOFTWARE_VERSION"] = k8sVersion
-		env["KAIROS_SOFTWARE_VERSION_PREFIX"] = string(config.DefaultConfig.KubernetesProvider)
+	versionInfo, err := getK8sInfo(log)
+	if err == nil && versionInfo.Provider != "" && versionInfo.Version != "" {
+		env["KAIROS_SOFTWARE_VERSION"] = versionInfo.Provider
```

Comment:
Does it still make sense to have a `KAIROS_SOFTWARE_VERSION` within the `/etc/kairos-release`? maybe we should let the provider be the one adding those env vars to the release file or maybe even have a file of their own?

Reply:
yeah not sure, mainly for compat reasons? Or to get aurora to read those and use those?